--- stage3/rootfs/etc/dbus-1/system.d/shairport-sync-mpris-policy.conf
+++ stage3/rootfs/etc/dbus-1/system.d/shairport-sync-mpris-policy.conf
@@ -4,7 +4,10 @@
           "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
 <busconfig>
 
-  <!-- Allow users "root" and "shairport-sync" to own the Shairport Sync MPRIS service -->
+  <!-- Allow users "root", "shairport-sync" and "pi" to own the Shairport Sync MPRIS service -->
+  <policy user="pi">
+    <allow own="org.mpris.MediaPlayer2.ShairportSync"/>
+  </policy>
   <policy user="shairport-sync">
     <allow own="org.mpris.MediaPlayer2.ShairportSync"/>
   </policy>
